src/contract_wrappers/registries/security_token_registry_wrapper.ts

```diff
@@ -810,7 +810,7 @@ export default class SecurityTokenRegistryWrapper extends ContractWrapper {
     if (registrationDate.getTime() === new Date(0).getTime()) {
       return true;
     }
-    if (!isDeployed && expiryDate.getTime() > Date.now()) {
+    if (!isDeployed && expiryDate.getTime() < Date.now()) {
```

Comment:
I think this is wrong. If `expiryDate.getTime()` is less than current time, the ticker is reserved, so it is not available.

Reply:
Reversed this change